docs/proposals/workload.md

```diff
@@ -0,0 +1,72 @@
+# Workload schema
+
+Example :
+
+```json
+{
+    "name": "test-workload",
+    "type": "container",
+    "executable": "nginx",
+    "ressources": {
+        "cpu": 1,
+        "ram": 2048,
+        "disk": 2,
+    },
+    "ports": [
+        "8080:80"
+    ]
+}
+```
+## name
+
+**type :** string
+
+Name of the workload, should be unique in the controller’s database.
+
+## type
+
+**type :** string
+
+Type of the workload :
+
+- `"container"`
+- `"vm"`
+- `"binary"`
+
+## executable
+
+**type :** string
+
+<!-- maybe change name -->
+
+The name of the container image , or an URI to the vm image or binary executable.
+
+## ressources
+
+**type :** object/struct
+
+Definition of the maximum (container, binary) or allocated (vm) ressources.
+
+### ressources.cpu
+
+**type :** TDB
+
+CPU power, unit is to be defined.
```

Comment:
This ressource could be represended in milliCPU. this unit represents the maximum portion of the CPU that could be use as a 1/1000 of a CPU.

`1 milliCPU = 1/1000 of a CPU`.

Reply:
I’ve set it as a float, are you okay with that ?

Reply:
No we concur to set all values in int to avoid casting and potential problems